crates/vsock/src/main.rs

```diff
@@ -43,45 +51,100 @@ struct VsockArgs {
     #[command(flatten)]
     param: Option<VsockParam>,
 
+    /// Configuration of the device corresponding to a guest in the form of comma separated key=value pairs.
```

Comment:
In order to avoid confusion with the config file, I'd say: "Device parameters corresponding to a VM in the form of ..."